app/controllers/admin/admin_users_controller.rb

```diff
@@ -1,52 +1,58 @@
 class Admin::AdminUsersController < Admin::AdminController
   before_filter :require_sysadmin
+  before_filter :find_user, only: %i[edit update destroy]
+
+  rescue_from AdminUser::CannotDeleteCurrentUser do
+    redirect_to admin_admin_users_url, alert: :user_is_current_user
+  end
+
+  rescue_from AdminUser::MustBeAtLeastOneAdminUser do
+    redirect_to admin_admin_users_url, alert: :user_count_is_too_low
+  end
 
   def index
-    @users = AdminUser.by_name.paginate(:page => params[:page], :per_page => 50)
+    @users = AdminUser.by_name.paginate(page: params[:page], per_page: 50)
   end
 
   def new
     @user = AdminUser.new
   end
 
   def create
-    @user = AdminUser.create(admin_user_params)
+    @user = AdminUser.new(admin_user_params)
+
     if @user.save
-      redirect_to admin_admin_users_url, notice: "User was successfully created"
+      redirect_to admin_admin_users_url, notice: :user_created
```

Comment:
There was talk of Welsh at some point and we got an email from Montreal saying they wanted to use the codebase which being in Quebec, Canada needs French and English.